cmd/bucket-replication.go

```diff
@@ -868,9 +868,12 @@ func NewReplicationPool(ctx context.Context, o ObjectLayer, opts replicationPool
 		replicaCh:       make(chan ReplicateObjectInfo, 100000),
 		replicaDeleteCh: make(chan DeletedObjectVersionInfo, 100000),
 		mrfReplicaCh:    make(chan ReplicateObjectInfo, 100000),
+		workerKillCh:    make(chan struct{}, opts.Workers),
+		mrfWorkerKillCh: make(chan struct{}, opts.FailedWorkers),
```

Comment:
Isn't this missing in the `AddMRFWorker()` select below?

Reply:
Thanks, fixed that.